registry/tests/appRoutes.spec.ts

```diff
@@ -244,6 +280,35 @@ describe(`Tests ${example.url}`, () => {
             }
         });
 
+        it('should create record with existed domainId', async () => {
+            const domainId = await createRouterDomain(example.routerDomain);
+
+            const exampleWithExistedDomainId = {
+                ...example.correct,
+                domainId,
+            };
+
+            let response = await request.post(example.url)
```

Comment:
this should be within another try/finally block

Reply:
you are right

Reply:
done